source/dbo/Stored Procedures/Monitoring/uspUpdateTableCounts.sql

```diff
@@ -0,0 +1,77 @@
+﻿CREATE PROCEDURE [dbo].[uspUpdateTableCounts]
+AS
+	INSERT INTO [TableCounts] (CountTime
+		,MigrationNotificationsViewCount
+		,MigrationMBovisAnimalExposureViewCount
+		,MigrationMBovisExposureToKnownCaseViewCount
+		,MigrationMBovisOccupationExposuresViewCount
+		,MigrationMBovisUnpasteurisedMilkConsumptionViewCount
+		,MigrationSocialContextAddressViewCount
+		,MigrationSocialContextVenueViewCount
+		,TransfersViewCount
+		,TreatmentOutcomesCount
+		,EtsNotificationsCount
+		,LtbrNotificationsCount
+		,ETS_NotificationCount
+		,LTBR_DiseasePeriodCount
+		,LTBR_PatientEpisodeCount
+		,NotificationClusterMatchCount
+		,NotificationSpecimenMatchCount
+		,EtsSpecimenMatchCount
+	)
+
+	SELECT
+		GETDATE() AS CountTime
```

Comment:
Could we use GETUTCDATE() here just so we know for sure what time it ran (not very important but I do tend to use GETUTCDATE() whenever possible)

Reply:
So they both give the time to the nearest millisecond, but `GETUTCDATE` does it in UTC and `GETDATE` in the local timezone. I agree using `GETUTCDATE` is a better idea, and I've switched that over